hwdb.d/60-evdev.hwdb

```diff
@@ -434,6 +434,13 @@ evdev:name:Synaptics TM3289-002:dmi:*svnLENOVO*:pvrThinkPadX1Carbon5th*
  EVDEV_ABS_35=::19
  EVDEV_ABS_36=::19
 
+# Lenove Thinkpad X1 Tablet Gen3.
```

Comment:
typo: "lenove". And no full stop please

Reply:
fixed. Other comments also contain full stop...

Reply:
looks like the ones that contain the full stop are the ones that end in `gen` which is obvs short for `generation` but that doesn't apply here with `gen3`. And `gen. 3` makes less sense, so getting rid of it makes the most sense IMO.

Reply:
Ok...  Fixed.